invenio_communities/communities/services/config.py

```diff
@@ -136,16 +126,9 @@ class CommunityServiceConfig(RecordServiceConfig, ConfiguratorMixin):
     ]
 
     # Service components
-    components = [
-        MetadataComponent,
-        CustomFieldsComponent,
-        PIDComponent,
-        RelationsComponent,
-        CommunityAccessComponent,
-        OwnershipComponent,
-        FeaturedCommunityComponent,
-        OAISetComponent,
-    ]
+    components = FromConfig(
```

Comment:
Better overwriting logic will come from a different issue https://github.com/inveniosoftware/invenio-communities/issues/914 currently in triage.